### PR TITLE
chore: update flamingo rotation manual

### PR DESCRIPTION
**Pull request by morph-dev**
### What was wrong?

Some information on the Flamingo manual seems to be outdated, most notably the "notes".

While I don't have access to the [notes](https://notes.ethereum.org/YAiNsmc8SSq05TwYdk-8Eg) and [checklist](https://notes.ethereum.org/?nav=overview&template=b35733cd-b374-4b79-bc57-f2bb58ee651e), Kolby told me that we weren't really using them for the last couple of months.

### How was it fixed?

Remove the references to the notes. If people think they were useful, should we consider restarting doing them? Should we consider alternatives?

I'm not sure what items are on the **checklist** (no access), but maybe we can put them on the Flamingo page directly? Can anybody share it with me?

### To-Do

- [x] Add checklist items directly into Flamingo page
- [ ] Clean up commit history and use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

### CODE REVIEW

**Comment by ogenev** [comment]
Here is the current checklist template but I think we need to simplify and improve it if we plan to use it:

# Weekly `trin` Flamingo Checklist

## Monday

Kickoff:
- [ ] Announce that you are on rotation in Discord
- [ ] Give yourself the Discord role `@trin-flamingo`
- [ ] Read handoff notes from previous rotation
- [ ] Give weekly summary update in all-Portal call
- [ ] Link this checklist into the weekly notes
- [ ] Pick day of week for deployment, discuss in #trin

Daily:
- [ ] Read Discord, especially for help requests or signs of network issues
- [ ] Monitor [Glados](http://glados.ethportal.net/content/) changes, and corrolate with releases (trin, glados, other clients?)
- [ ] Monitor [portal-hive](https://portal-hive.ethdevops.io/) changes
  - [ ] Check the dates, did all test suites run on the previous cycle?
  - [ ] For each suite, did the expected number of tests run?
  - [ ] Did trin start failing any tests? (While other clients are catching up, focus on "trin,trin" tests)
  - [ ] If trin failing, create rotation issue to pursue
- [ ] Look for [inspiration for Flamingo projects](https://ethereum.github.io/trin/developers/contributing/rotation/index.html#maintenance-inspiration)
- [ ] Update week's notes with progress & plans

## Tuesday

- [ ] Read Discord, especially for help requests or signs of network issues
- [ ] Monitor [Glados](http://glados.ethportal.net/content/) changes, and corrolate with releases (trin, glados, other clients?)
- [ ] Monitor [portal-hive](https://portal-hive.ethdevops.io/) changes
  - [ ] Check the dates, did all test suites run on the previous cycle?
  - [ ] For each suite, did the expected number of tests run?
  - [ ] Did trin start failing any tests? (While other clients are catching up, focus on "trin,trin" tests)
  - [ ] If trin failing, create rotation issue to pursue
- [ ] Look for [inspiration for Flamingo projects](https://ethereum.github.io/trin/developers/contributing/rotation/index.html#maintenance-inspiration)
- [ ] Update week's notes with progress & plans

## Wednesday

- [ ] Read Discord, especially for help requests or signs of network issues
- [ ] Monitor [Glados](http://glados.ethportal.net/content/) changes, and corrolate with releases (trin, glados, other clients?)
- [ ] Monitor [portal-hive](https://portal-hive.ethdevops.io/) changes
  - [ ] Check the dates, did all test suites run on the previous cycle?
  - [ ] For each suite, did the expected number of tests run?
  - [ ] Did trin start failing any tests? (While other clients are catching up, focus on "trin,trin" tests)
  - [ ] If trin failing, create rotation issue to pursue
- [ ] Look for [inspiration for Flamingo projects](https://ethereum.github.io/trin/developers/contributing/rotation/index.html#maintenance-inspiration)
- [ ] Update week's notes with progress & plans

## Thursday

- [ ] **Unique**: Identify the next Flamingo
- [ ] Read Discord, especially for help requests or signs of network issues
- [ ] Monitor [Glados](http://glados.ethportal.net/content/) changes, and corrolate with releases (trin, glados, other clients?)
- [ ] Monitor [portal-hive](https://portal-hive.ethdevops.io/) changes
  - [ ] Check the dates, did all test suites run on the previous cycle?
  - [ ] For each suite, did the expected number of tests run?
  - [ ] Did trin start failing any tests? (While other clients are catching up, focus on "trin,trin" tests)
  - [ ] If trin failing, create rotation issue to pursue
- [ ] Look for [inspiration for Flamingo projects](https://ethereum.github.io/trin/developers/contributing/rotation/index.html#maintenance-inspiration)
- [ ] Update week's notes with progress & plans

## Friday

Wrap-up:

- [ ] Haven't deployed yet? Oops, a bit late. Get it done as early as possible.
- [ ] Comment on the checklist template to add/update/delete anything?

Daily:
- [ ] Read Discord, especially for help requests or signs of network issues
- [ ] Monitor [Glados](http://glados.ethportal.net/content/) changes, and corrolate with releases (trin, glados, other clients?)
- [ ] Monitor [portal-hive](https://portal-hive.ethdevops.io/) changes
  - [ ] Check the dates, did all test suites run on the previous cycle?
  - [ ] For each suite, did the expected number of tests run?
  - [ ] Did trin start failing any tests? (While other clients are catching up, focus on "trin,trin" tests)
  - [ ] If trin failing, create rotation issue to pursue
- [ ] Look for [inspiration for Flamingo projects](https://ethereum.github.io/trin/developers/contributing/rotation/index.html#maintenance-inspiration)
- [ ] Update week's notes with progress & plans

Handoff:
- [ ] Add next week to top of Flamingo notes, with next person on rotation
- [ ] Final review of Flamingo notes, summarize for readability
- [ ] Send the Flamingo notes link to the next Flamingo

**Comment by morph-dev** [comment]
Thanks. I simplified and added checklist.

Another topic that I would like to discuss. Should we restart using [Flamingo rotation](https://notes.ethereum.org/@njgheorghita/r1angO2lT) document?
I think it's really useful to have long term overview, especially when in comes to planning vacations and such

File: book/src/developers/contributing/rotation/index.md [hunk 1]
```diff
@@ -66,22 +90,15 @@ When you get to the end of your checklist, here are ideas for what to work on ne
...
-Double check the [Flamingo schedule](https://notes.ethereum.org/@njgheorghita/r1angO2lT) and make sure you're available for your next rotation. If not, please switch with somebody asap.
-
-Think through: what kinds of things do you wish were on the notes that you got on Monday?
+Especially while we continue to develop the procedure, try to be available the following Monday to help the subsequent Flamingo transition in, and get started.
 
-After a while, everyone should have the Flamingo Notes link. Sending the link anyway helps accomplish a few things:
-1. Signal to the next Flamingo that you're done writing up the notes
-2. Give the next Flamingo a reminder that their turn is coming up
-3. A failsafe to make sure we didn't forget to pick a Flamingo for the following week
+Think through: what kinds of things do you think should on the checklist?
```
Comment (njgheorghita) [inline book/src/developers/contributing/rotation/index.md:99]: `do you think should be on the checklist`

File: book/src/developers/contributing/rotation/index.md [hunk 2]
```diff
@@ -66,22 +90,15 @@ When you get to the end of your checklist, here are ideas for what to work on ne
...
 
-Especially while we continue to develop the procedure, try to be available the following Monday to help the subsequent Flamingo transition in, and get started.
+Double check the [Flamingo schedule](https://notes.ethereum.org/@njgheorghita/r1angO2lT) and make sure you're available for your next rotation. If not, please switch with somebody asap.
```
Comment (njgheorghita) [inline book/src/developers/contributing/rotation/index.md:101]: This is also another remnant that isn't actively maintained. 
- I can update the schedule
- Update this to say something like... "estimate when you'll be flamingo next and make sure you're available..."

**Review by njgheorghita — APPROVED** [review]
🦩

**Comment by ogenev** [comment]
> Another topic that I would like to discuss. Should we restart using [Flamingo rotation](https://notes.ethereum.org/@njgheorghita/r1angO2lT) document?

I don't have access to this document, @njgheorghita  if you are the owner, could you add me? :)

**Review by ogenev — APPROVED** [review]

File: book/src/developers/contributing/rotation/index.md (hunk 2 above)
Comment (morph-dev) [reply book/src/developers/contributing/rotation/index.md:101]: As I said in the other comment, I think we should restart using it.

And there shouldn't be "estimation" on when next shift should come, meaning that it should always be planned at least one-two months in the future.

I volunteer to take responsibility and make sure we always have a schedule ahead. Others just have to be sure they keep it updated when they change shifts.

**Comment by njgheorghita** [comment]
@ogenev @morph-dev Yup, I'm happy to open access to that document... but maybe there's a simpler place to host this message? Like an open issue in this repo or something else more github-y

**Comment by KolbyML** [comment]
I am not a fan of having a constantly updated flamingo whose turn it is document in the trin repo, if we want a repo with a md document we should make a flamingo repo for it or a discord channel

File: book/src/developers/contributing/rotation/index.md [hunk 3]
```diff
@@ -20,31 +21,54 @@ There are some daily tasks, and some transition tasks: Kickoff on Monday & Hando
...
 ### Daily
 
 #### Checklist
 
-Every day, go down the checklist for that day of the week, resolve items and check them off. If you think of new daily things to add to the checklist, comment on the template to suggest it.
+Every day, go down the daily and items for that day of the week. If you think of new daily things to add to the checklist, create a PR.
+
+- **Daily**
+  - Read Discord, especially for help requests or signs of network issues
+  - Monitor [Glados](http://glados.ethportal.net/) changes, and correlate with releases (trin, glados, other clients?)
```
Comment (carver) [inline book/src/developers/contributing/rotation/index.md:42]:
```suggestion
  - Monitor [Glados](https://glados.ethdevops.io/) changes, and correlate with releases (trin, glados, other clients?)
```

I know this url was out of date elsewhere in the docs. Might as well update it while you're tweaking.

**Review by carver — APPROVED** [review]
🚢

**Comment by morph-dev** [comment]
I'm merging this one. If anybody has additional followup, I will make another PR.

**Pull request merged**